web/modules/custom/farm_nfa/src/Form/ForestPlanGfwForm.php

```diff
@@ -73,7 +73,7 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#map_settings' => [
         'plan' => $this->routeMatch->getRawParameter('plan'),
         'host' => $this->request->getHost(),
-        'scheme' => $this->request->getScheme(),
+        'scheme' => $this->request->headers->get('X-Forwarded-Proto') === 'https' ? 'https' : 'http',
```

Comment:
I think if you do `//` instead of http/https the browser will autodetect and you won't need this